tools/FindEigen3.cmake

```diff
@@ -64,6 +64,9 @@ if(EIGEN3_INCLUDE_DIR)
   set(EIGEN3_FOUND ${EIGEN3_VERSION_OK})
 
 else(EIGEN3_INCLUDE_DIR)
+  if(NOT DEFINED KDE4_INCLUDE_DIR)
```

Comment:
Ah, right, we can't use `possibly_uninitialized` here? Ah, too bad. So the solution is already less useful :-)